zenoh/src/api/session.rs

```diff
@@ -657,6 +657,28 @@ impl Session {
     pub fn config(&self) -> &Notifier<Config> {
         self.runtime.config()
     }
+
+    /// Get a new Timestamp from a Zenoh session [`Session`](Session).
+    ///
+    /// The returned timestamp has the current time, with the Session's runtime ZenohID
+    ///
+    /// # Examples
+    /// ### Read current zenoh configuration
+    /// ```
+    /// # #[tokio::main]
+    /// # async fn main() {
+    /// use zenoh::prelude::*;
+    ///
+    /// let session = zenoh::open(zenoh::config::peer()).await.unwrap();
+    /// let timestamp = session.new_timestamp();
+    /// # }
+    /// ```
+    pub fn new_timestamp(&self) -> Timestamp {
+        let id = self.runtime.zid();
+        // TODO: Should we make this an Result return type ?
```

Comment:
`duration_since(UNIX_EPOCH)` should never fail since `SystemTime` is not allowed to be earlier than `UNIX_EPOCH`.